content/docs/dvclive/how-it-works.md

```diff
@@ -73,43 +73,54 @@ model.pt.dvc
 
 ## Track the results
 
-DVCLive expects each run to be tracked by Git, so it will save each run to the
-same path and overwrite the results each time. Include
+### Git integration
+
+Unlike other experiment trackers, DVCLive relies on Git to track the [directory]
```

Comment:
My 2cs: I think track results can start with a bit basic stuff and something that I think more people can relate to / understands faster.

1.that we can track them in VS Code and Studio
2.may be ways to compare experiments, or just experiments, or tracking experiments - that where we can go into Git concept to a certain degree and large files, etc (even though I still think we need 

The biggest issues with explanation is that people don't expect it / can't most likely even understand why we put it here until they hit some issues. 

May be another idea - "DVCLive vs other trackers: important workflow details".

Reply:
Renamed from "Track the results" to "Git and DVC integration" and introduced it by explaining that this differentiates it from other experiment trackers.